### PR TITLE
FEAT : Comment, follow, unfollow, update-profile

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
- Add comment  / del comment
- Edit comment 
- upvote / downvote comment
- follow / unfollow user
- update / edit user profile

### CODE REVIEW

File: src/frontend/features/user/singleUserSlice.js [hunk 1]
```diff
@@ -44,6 +44,27 @@ const getUserAllPostsById = createAsyncThunk(
...
+        { userData },
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      return data;
+    } catch (err) {
+      console.error(err);
```
Comment (horsemaker) [inline src/frontend/features/user/singleUserSlice.js]: You could remove the console here.

**Review by horsemaker — APPROVED** [review]
Everything looks clean & I absolutely love the UI! 👌

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean and well structured

**Review by saketkum11 — APPROVED** [review]
- Clean UI
-  code looks clean.

**Pull request merged**